packages/babel-preset-peregrine/lib/index.js

```diff
@@ -0,0 +1,73 @@
+const plugins = [
+    /**
+     * See:
+     * https://babeljs.io/docs/en/babel-plugin-proposal-class-properties
+     * https://babeljs.io/docs/en/babel-plugin-proposal-object-rest-spread
+     * https://babeljs.io/docs/en/next/babel-plugin-syntax-dynamic-import.html
+     * https://babeljs.io/docs/en/next/babel-plugin-syntax-jsx.html
+     * https://babeljs.io/docs/en/babel-plugin-transform-react-jsx
+     * https://www.npmjs.com/package/babel-plugin-graphql-tag
+     */
+    ['@babel/plugin-proposal-class-properties'],
+    ['@babel/plugin-proposal-object-rest-spread'],
+    ['@babel/plugin-syntax-dynamic-import'],
+    ['@babel/plugin-syntax-jsx'],
+    ['@babel/plugin-transform-react-jsx'],
+    ['babel-plugin-graphql-tag']
+];
+
+const config = (api, opts = {}) => {
+    // Different environments, different settings for preset-env.
+    const targets = Object.assign(
+        {},
+        {
+            // For maximum recompile speed:
+            dev: 'last 2 Chrome versions',
+            // A consuming package can optionally provide a list of browsers;
+            // otherwise, use the defaults.
+            prod: require('./browserslist'),
+            // The Jest test runner provides a synthetic DOM, but  not a real
+            // browser environment; instead, it's Node.
+            test: 'node 8'
```

Comment:
Why the switch from `node 10`?

Reply:
Now.sh lambda only supports Node 8 still.